genai-perf/genai_perf/inputs/retrievers/synthetic_data_retriever.py

```diff
@@ -58,16 +58,17 @@ def retrieve_data(self) -> GenericDataset:
                 self.config.prompt_tokens_mean,
                 self.config.prompt_tokens_stddev,
             )
-            row.texts.append(prompt)
+            for _ in range(self.config.batch_size_text):
+                row.texts.append(prompt)
 
-            if self.config.output_format == OutputFormat.OPENAI_VISION:
```

Comment:
I removed this format-specific code. We don't want format-specific code in the retrievers. This would prevent new image converters from generating image data via the synthetic route.

There will be latency introduced by generating both texts and images. I created